### PR TITLE
fix readme & remove unnecessary helm templates

### PR DESCRIPTION
**Pull request by rasheedamir**
- remove unnecessary helm templates
- fix readme

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-7-0bd03cfa`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-7-6d33db7f`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull stakater/workshop-operator:SNAPSHOT-PR-7-a832bb3d`

File: README.adoc [hunk 1]
```diff
@@ -57,6 +57,6 @@ oc apply -n workshop-infra -f config/samples/workshop_v1_cloud_native_workshop.y
 
 [source,bash]
 ----
-make docker-build IMG=quay.io/stakater/workshop-operator
-make docker-push IMG=quay.io/stakater/workshop-operator
+make docker-build IMG=docker.io/stakater/workshop-operator
```
Comment (MaazMS) [inline README.adoc:60]: update image url

File: README.adoc [hunk 2]
```diff
@@ -57,6 +57,6 @@ oc apply -n workshop-infra -f config/samples/workshop_v1_cloud_native_workshop.y
...
+make docker-push IMG=docker.io/stakater/workshop-operator
```
Comment (MaazMS) [inline README.adoc:61]: update image url

**Review by MaazMS — APPROVED** [review]

**Pull request merged**